src/index.html

```diff
@@ -0,0 +1,240 @@
+	-->
+	<script>(function(h){h.className = h.className.replace('no-js', 'js')})(document.documentElement)</script>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js"></script>
+  <link href="https://use.fontawesome.com/releases/v5.0.4/css/all.css" rel="stylesheet">
+
+	<!-- Place favicon.ico and apple-touch-icon.png in the root directory. -->
+
+	<!-- Place anything custom after this. But put JS to the end of body. -->
+</head>
+
+<body>
+
+	<!-- begin: content -->
+  <header class="navigation">
+    <div class="nav-container">
+       <div class="brand">
+          <a class="navbar-brand" href="#">
+          <img src="assets/img/Baumeister-Roboter.png" alt="">
+          </a>
+       </div>
+       <nav>
+          <div class="nav-mobile"><a id="nav-toggle" href="#!"><span></span></a></div>
+          <ul class="nav-list" id="navlist">
+             <li>
+                <a href="#!">Start</a>
+             </li>
+             <li>
+                <a href="#anchor-about">About</a>
+             </li>
+             <li>
+                <a href="#anchor-iforp">iFORP</a>
+             </li>
+             <li>
+                <a href="#anchor-screencasts">Screencasts</a>
+             </li>
+             <li>
+                <a href="#anchor-install">Install</a>
+             </li>
+             <li>
+                <a href="#anchor-socials">Socials</a>
+             </li>
+          </ul>
+       </nav>
+    </div>
+ </header>
+ <div class="container-fluid">
+ <div class="row robot-header white">
+       <div class="robot-header-left col-lg-6 col-md-12 col-sm-12 col-xs-12">
+          <h1>Baumeister</h1>
+          <div class="robot-flying">
+             <img class="hover_wide" src="assets/img/Baumeister-Roboter.png" alt="Baumeister-Roboter">
+          </div>
+          <div class="robot-burst">
+             <img class="big_boost hover_short1" src="assets/img/big_boost.png" alt="">
+             <img class="middle_boost hover_short2" src="assets/img/middle_boost.png" alt="">
+             <img class="small_boost hover_short3" src="assets/img/small_boost.png" alt="">
+          </div>
+       </div>
+       <div class="robot-header-right col-lg-6 col-md-12 col-sm-12 col-xs-12">
+          <h1>The frontend build workflow for your needs</h1>
+          <p>
+             The aim of this project is to help you to build your things.
+             <br>
+             From Bootstrap themes over static websites to single page applications.
+          </p>
+          <div><a class="button" href="#anchor-install" >Install Now</a></div>
+    </div>
+ </div>
+ <div class="row our-patners gray">
+       <div class="patner bmbf col-lg-2 offset-lg-2 col-md-2 offset-md-1 col-ms-12 col-xs-12">
```

Comment:
Die Klasse `patner` umbennen in `partner`.

Reply:
Taucht an mehreren Stellen auf.

Reply:
Done